docs/dataset-therapeutics.md

```diff
@@ -0,0 +1,34 @@
+The Therapeutics dataset contains information on ...
+
+!!! warning
+    Therapeutics data can only be used ...
+    Please contact your co-pilot, or <team@opensafely.org> if you have any questions.
+
+The Therapeutics dataset is...
+
+## Metadata
+
+* **Data provider** BlueTeq
+* **Participation / Coverage** ...
+* **Provenance** ...
+* **Update frequency in OpenSAFELY** Approximately weekly.
+* **Delay between event occurring and event appearing in OpenSAFELY** Approximately 1-2 weeks.
+* **Collected information** Treatment start date; therapeutic intervention; COVID indication, current status, risk group, region
+ 
+
+## Overview
+...
+
+
+## More Information
+
+...Links to any other sources
+
+## Datasource-specific glossary
+
+...Any specific terms to define
+* **Term** description
+Each treatment is a row.  Patients should only appear once per episode treated, but may legitimately appear more than once if they received a second treatment for a separate episode.
```

Comment:
This is almost certainly not right, but we probably want to describe what multiple rows per patient should mean?

Reply:
Each row is one completed form for one course of treatment. The patient may or may not have actually received the treatment or completed the course (but we assume that they usually do). They may have another form completed for another treatment, either because it was decided to give them a different treatment, or for some other reason. They may in theory also have another form completed some months later for another instance of infection.